xmipp

```diff
@@ -1166,6 +1167,7 @@ def exitProgram(error=0, status=None):
             show_output=False, show_command=False, log=log)
     except Exception as e:
         pass
+    sendApiPOST(error, XMIPP_VERSION)
```

Comment:
I would defer the import here and put it on a try-catch structure for extra safety. In addition, the try-catch block of `sendApiPOST` would be no longer needed.

```
try:
        from scripts.API import sendApiPOST
        sendApiPOST(error, XMIPP_VERSION)
except Exception as e:
        pass
```

Reply:
The import should never fail once it's done properly (if it's not, it should never work), so I see no point in adding it into the try-except block, which, as a good practice, should contain the least amount of code possible inside it.